tests/tck-build-logic/src/main/groovy/org.graalvm.internal.tck-harness.gradle

```diff
@@ -124,7 +124,11 @@ Provider<Task> generateMatrixDiffCoordinates = tasks.register("generateMatrixDif
             def matrix = [
                 "coordinates": ["No matches found!"]
             ]
+
             matrix.putAll(matrixDefault)
+            matrix.version.add("21");
```

Comment:
Could you add a comment here why we do this? Also, I think we want to add `22` now that it's out :)